draft-ietf-quic-recovery.md

```diff
@@ -753,6 +753,12 @@ delivery of ACK frames is important for efficient loss recovery. Packets
 containing only ACK frames should therefore not be paced, to avoid delaying
 their delivery to the peer.
 
+Sending multiple packets into the network without any delay between them
+creates a packet burst that might cause short-term congestion and losses.
+Implementations MUST either use pacing or limit such bursts to minimum
+of 10 * kMaxDatagramSize and max(2* kMaxDatagramSize, 14720)), the same
+as the recommended initial congestion window.
+
```

Comment:
Sorry again late here... To we recommend a fixed value of 10 packets or do we recommend to have the burst limit the same as the initial window? E.g. if we decide in future to increase our recommendation for the initial window, should that also increase the recommendation for the burst limit? I guess to be really smart about this, you would need to say that you set the burst but initial window but decrease if you see tail loss in the initial burst (but that might be an overkill).

Reply:
Or you could say that there are different initial window and burst limits, which is probably OK.  I don't think that we need to be that smart though.

Reply:
That's a good point.  We could swap the sentence and say "same as the initial congestion window, which is recommended to be ..." if you think it's preferable.  But I'm also not sure it matters much.

Reply:
I think that explicitly saying "Initial congestion window" here as the primary target of the MUST requirement would be better.

Reply:
Fine for me.

Reply:
PR created, PTAL: https://github.com/quicwg/base-drafts/pull/3160